priv/templates/list/list-item-kg.event.tpl

```diff
@@ -1,8 +1,11 @@
 
 {% if not (exclude and id.id|member:exclude) %}
 	{% with id.o.hasbanner[1]|default:id.depiction.id|default:m.rsc.fallback.id as dep %}
-	<li class="list-item-kg--event" style="background-image: url('{% image_url dep mediaclass='list-image' crop=dep.crop_center %}');">
+	<li class="list-item-kg--event" style="background-image: url('{% image_url dep mediaclass='list-image' crop=dep.crop_center %}'); position: relative; overflow: hidden;">
 		<a href="{{ id.page_url }}">
+			{% if id.date_start < now %}
+				<div class="event-status__past">Geweest</div>
+			{% endif %}
```

Comment:
One suggestion, I'd change this to:
```suggestion
			{% if id.date_end < now %}
				<div class="event-status__past">Geweest</div>
			{% endif %}
```
otherwise the ribbon will appear also on events that are still ongoing (when `date_start` has passed, but `date_end` hasn't yet).